### PR TITLE
fix(js): Truncate workflow name and center empty notifications text

### PR DESCRIPTION
**Pull request by desiprisg**
### Screenshots
Screenshot 2024-11-25 at 1 56 38 PM
Screenshot 2024-11-25 at 1 58 21 PM

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/js/src/ui/components/elements/Preferences/Preferences.tsx [hunk 1]
```diff
@@ -148,17 +148,14 @@ const PreferencesRow = (props: {
...
           class={style(
             'workflowLabelContainer',
-            'nt-flex nt-justify-between nt-flex-nowrap nt-self-stretch nt-cursor-pointer nt-items-center'
+            'nt-flex nt-justify-between nt-flex-nowrap nt-self-stretch nt-cursor-pointer nt-items-center nt-overflow-hidden'
           )}
           onClick={() => setIsOpen((prev) => !prev)}
           data-open={isOpen()}
         >
-          <div>
+          <div style={{ overflow: 'hidden' }}>
```
Comment (rifont) [inline packages/js/src/ui/components/elements/Preferences/Preferences.tsx]: If someone tried to override the default behavior via the appearance prop classnames, wouldn't this take precedence and prevent them from modifying behavior?

Comment (desiprisg) [reply packages/js/src/ui/components/elements/Preferences/Preferences.tsx]: Correct. Don't see any case for this to happen now though. We have no descriptor here so you can't override anyway. Can you come up with a good name for it?

Comment (rifont) [reply packages/js/src/ui/components/elements/Preferences/Preferences.tsx]: I don't think there's a particularly strong case either, but still good to keep it flexible. Here are some options:

1. workflowLabelSection
2. workflowLabelHeader
3. workflowLabelDisplay
4. workflowLabelPanel

Personally I would go with `workflowLabelHeader`.

Comment (desiprisg) [reply packages/js/src/ui/components/elements/Preferences/Preferences.tsx]: Done! Nice suggestion.

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**